packages/iris-grid/src/IrisGridTheme.module.scss

```diff
@@ -24,13 +24,13 @@ $selection-outline-casing-color: $gray-900;
   row-hover-bg: mix($gray-600, $gray-700, 50%);
   selected-row-hover-bg: rgba($blue, 0.25);
 
-  scroll-bar-bg: $gray-700;
-  scroll-bar-hover-bg: $gray-600;
-  scroll-bar-casing-color: $black;
-  scroll-bar-corner-color: $gray-800;
-  scroll-bar-color: $gray-400;
-  scroll-bar-hover-color: $gray-300;
-  scroll-bar-active-color: $gray-200;
+  scroll-bar-bg: $gray-850;
+  scroll-bar-hover-bg: $gray-800;
+  scroll-bar-casing-color: $gray-600;
+  scroll-bar-corner-color: $gray-850;
+  scroll-bar-color: $gray-500;
+  scroll-bar-hover-color: $gray-400;
+  scroll-bar-active-color: $gray-300;
```

Comment:
This gray isn't matching the native scroll bar gray when dragging in Chrome

Reply:
I am going to say that is acceptable, I don't want to introduce a new color step just for the grid active state, and since they are all constructed differently, this is close enough.